apps/pipelines/tests/test_parallel_nodes.py

```diff
@@ -62,6 +62,46 @@ def test_parallel_branch_pipeline(pipeline, experiment_session):
     assert output == expected_output
 
 
+@django_db_with_data(available_apps=("apps.service_providers",))
+def test_parallel_branch_with_merge(pipeline, experiment_session):
+    """
+    Illustrate and validate what happens with parallel branches with an aggregator node.
+
+    start -> A -----> D -> end
```

Comment:
Am I right in saying that we're always going to be returning the output of whichever node rejoins (in this case D) last?